### ISSUE TITLE
Determine & Set Encoding

### ISSUE THREAD

**Issue by YvesR** [jacobslusser/ScintillaNET#296]
Hello,

is there a way to set the encoding format of a current loaded text for a editor?
Something like `TextArea.Encoding = Encoding.UTF8;`.

And / Or a property that returns the current encoding from the textarea?

**Comment by jacobslusser** [comment]
TL;DR: No.

Technically, the native Scintilla component actually supports other encoding forms, but ScintillaNET v3+ does not because this just trips-up developers. If you browse through the issues list here and some of our documentation you'll see my many explanations why, but the short answer is that developers expect it to work like the standard `Windows.Forms.TextBox` does and that means working with `string` and not `byte[]` as one would have to if you were using encodings.

Generally speaking this is not an issue because a developer can use one of the many `Encoding.Xxx` classes to get their text from any encoding format it is originally in and convert it to `string` before handing it to ScintillaNET.

**Comment by YvesR** [comment]
So ScintillaNET internal always use UTF-8 ?

**Comment by jacobslusser** [comment]
Yup. :)
